lib/helpers.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) 2022-2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  ECDSA_CURVE,
+  MULTICODEC_P256_PUBLIC_KEY_HEADER,
+  MULTICODEC_P256_SECRET_KEY_HEADER,
+  MULTICODEC_P384_PUBLIC_KEY_HEADER,
+  MULTICODEC_P384_SECRET_KEY_HEADER,
+  MULTICODEC_P521_PUBLIC_KEY_HEADER,
+  MULTICODEC_P521_SECRET_KEY_HEADER
+} from './constants.js';
+
+// retrieves name of appropriate ECDSA curve
+export function getNamedCurve({publicMultikey}) {
+  if(publicMultikey[0] === MULTICODEC_P256_PUBLIC_KEY_HEADER[0]) {
+    if(publicMultikey[1] === MULTICODEC_P256_PUBLIC_KEY_HEADER[1]) {
```

Comment:
These can all be made into compound AND statements:

```suggestion
  if(publicMultikey[0] === MULTICODEC_P256_PUBLIC_KEY_HEADER[0] &&
    publicMultikey[1] === MULTICODEC_P256_PUBLIC_KEY_HEADER[1]) {
```